apps/web/composables/useCategoryTemplate/types.ts

```diff
@@ -9,7 +9,7 @@ export interface UseCategoryTemplateState {
 
 export type FetchCategoryTemplate = (categoryId: number) => Promise<void>;
 export type SaveBlocks = (identifier: string | number, type: string, content: string) => Promise<void>;
-export type GetBlocks = (identifier: number | string, type: string) => Promise<void>;
+export type GetBlocks = (identifier: number | string, type: string, block?: string) => Promise<void>;
```

Comment:
```suggestion
export type GetBlocks = (identifier: number | string, type: string, blocks?: string) => Promise<void>;
```
I think the method supports a comma-separated list of blocks, doesn't it? If so, we should use the plural here.